tools/topology/sof-glk-src-max98357.m4

```diff
@@ -0,0 +1,90 @@
+#
+# Topology for GeminiLake with MAX98357.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include bxt DSP configuration
+include(`platform/intel/bxt.m4')
+
+#
+# Define the pipelines
+#
+# PCM0  ----> SRC (pipe 1)   -----> SSP1 (speaker - maxim98357a, BE link 0)
+#
+
+# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
```

Comment:
@zhuyingjiang this doesnt match the commit message? this is just volume playback pipeline.

Reply:
@ranj063  oops, I will double check, should because of submit the wrong branch.

Reply:
@ranj063 Strange things, I checked that both `PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4` and `PIPELINE_PCM_ADD(sof/pipe-src-playback.m4` works.
Anyway, I will update this to SRC and then check why volume still work. @singalsu do you have any idea?

Reply:
@zhuyingjiang we dont need src pipeline for GLK. so let drop this PR